rspace/src/main/scala/coop/rchain/rspace/IStore.scala

```diff
@@ -19,6 +23,8 @@ trait IStore[C, P, A, K] {
     */
   private[rspace] type T
 
+  private[rspace] type TRIE_TXN
```

Comment:
I guess if we're going to open this door I'm thinking something like:

```scala
private[rspace] type Transaction
 	 
private[rspace] type TrieTransaction
```

The [Scala Style Guide](https://docs.scala-lang.org/style/naming-conventions.html) does not make any explicit suggestions about naming type members, but given some of the other directions, I'd be willing to infer that upper camel case makes the most sense here.

OTOH, I definitely don't like upper snake case, nor do I like the asymmetry between `T` and `TXN`.  I also think it's probably best to avoid using `Txn` as that's already claimed by lmdbjava.

Reply:
:+1: from me, as this would be consistent with:
https://docs.scala-lang.org/style/naming-conventions.html#type-parameters-generics